### DISCUSSION TITLE
Possible to change system naming identifiers? (eg `+config.h.js``

### DISCUSSION THREAD

**Discussion by damassi** [vikejs/vike#1442] [Discussion]
Hi! 

i was wondering if its possible to rename these file identifiers at the config level? Eg:
- +config.h.js` => config.js
- +Page.js` => page.js

I understand the purpose, but it would be nice for those who appreciate minimalism to be able to configure these. 

Another alternative would be to simply go with the `_` convention, similar to next.

**Comment by brillout** [comment]
What's your problem with the `+` sign?

**Reply by bravius** [reply]
Just curious, what is the significance of the "plus files" + prefix? Is this to group or distinguish them as framework files?

**Reply by brillout** [reply]
https://github.com/vikejs/vike/discussions/1461#discussioncomment-8267342
